discount/src/main/asciidoc/index.adoc

```diff
@@ -0,0 +1,89 @@
+= Discount microservice
+Andy Wilkinson; Jay Bryant
```

Comment:
Andy Wilkinson; Jay Bryant?

Reply:
You are looking at the old version. I have already deleted this copy-paste